cycling-chd-paper/cycling-chd-paper.Rmd

```diff
@@ -73,6 +73,10 @@ active transport at population level in order to inform policy making
 relationship, including socioeconomic status, smoking prevalence, (gender,
 ethnicity and urbanity)
 
+Is the scope just Coronary heart disease? Not cardiovascular disease? Stroke, etc may be interesting in the same vein and may be quite easy to do at the same time
+
+May need to take other confounding variables into account - diet?
```

Comment:
Sure - sorry still trying to figure it out

thanks - let me know what you think


> On 16 May 2016, at 09:28, Robin <notifications@github.com> wrote:
> 
> In cycling-chd-paper/cycling-chd-paper.Rmd <https://github.com/Robinlovelace/cycling-chd/pull/8#discussion_r63322772>:
> 
> > @@ -73,6 +73,10 @@ active transport at population level in order to inform policy making
> >  relationship, including socioeconomic status, smoking prevalence, (gender,
> >  ethnicity and urbanity)
> >  
> > +Is the scope just Coronary heart disease? Not cardiovascular disease? Stroke, etc may be interesting in the same vein and may be quite easy to do at the same time
> > +
> > +May need to take other confounding variables into account - diet?
> Hi Yann maybe - we can have this discussion via email but suggest in the main text of the paper is not the ideal placee. FYI to add comments use this <!-- symbol like so:
> 
> <!-- This is a comment -->
> <!--
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/Robinlovelace/cycling-chd/pull/8/files/1e8073d055645e831371a6b2e33a6207e9729a54#r63322772>